valhalla/config.h

```diff
@@ -37,6 +38,22 @@
 /* Define to the version of this package. */
 #define PACKAGE_VERSION VALHALLA_VERSION
 
+#if !defined(VALHALLA_SOURCE_DIR)
+#define VALHALLA_SOURCE_DIR
+#endif
+
+inline std::string get_relative_file_path(std::string file_path) {
+  std::string valhalla_dir_str = VALHALLA_STRINGIZE(VALHALLA_SOURCE_DIR);
+  size_t len = valhalla_dir_str.length();
+  return file_path.substr(len + 1);
```

Comment:
cant we just move these few lines inside of the macro in scoped timer? id rather not increase the stuff in our public headers if we dont have to